generator/template/ready-to-use-section-types/components/icons/Copy.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
no asset should be shipped with this library, instead it should be hosted somewhere else and be used through css so it can be easily customized

applies for all images